### ISSUE TITLE
Multiselect

### ISSUE THREAD

**Issue by NilakArgento** [rossmartin/cordova-plugin-instagram-assets-picker#3]
Hi can you add options like multiple select and select media(image,video) from specific folder ?!

**Comment by rossmartin** [comment]
I have considered allowing to select media from a specific album in the photos library, this is a great idea thanks.

I'm not quite sure how to implement multiple select unless you are for sure not cropping immediately after the selection.

**Comment by NilakArgento** [comment]
of course,  if multiple select is activated then cropping should be unavailable)

**Comment by JFusco** [comment]
Thanks for creating this plugin.  I definitely have a need for multiselect for videos and image files - I'm mostly after the functionality that stops compression on video files when selected.  Have you had a chance to work on this functionality? If not, would you accept a pull request if we figure it out?

**Comment by rossmartin** [comment]
@JFusco Thanks I haven't had a chance to work on this yet.  Pull requests are definitely welcome.
